schemas/task.yml

```diff
@@ -187,7 +193,6 @@ required:
   - provisionerId
   - workerType
   - schedulerId
-  - taskGroupId
```

Comment:
> Also `priority` isn't required in the createTask call... There is a big difference in what is required.

It isn't required, but there is a default set, so should be ok.

In any case, I think I'll try to split it out so we have different required fields for inputs and outputs, but share the same data structure...